config/locales/input_objects/confirm_action.yml

```diff
@@ -1,7 +1,8 @@
+---
 en:
   helpers:
     label:
       confirm_action_input:
         options:
-          yes: "Yes"
-          no: "No"
+          'false': 'No'
+          'true': 'Yes'
```

Comment:
I'm not sure about this change 🤔

Reply:
It did - I think because YAML treats `yes` and `no` as equivalent to `true` and `false`. I think this is easier to read and still OK by i18n-tasks:
```
'no': 'No'
'yes': 'Yes'
```
so I'll update it

Reply:
Done